### ISSUE TITLE
App is not opening on Windows Startup

### ISSUE THREAD

**Issue by juniopereirab** [Izurii/easy-auto-launch#4]
- The target platform(s) the problem occurs on (Linux, Mac, Mac App Store, and or Windows):
Microsoft Windows 10 Pro
Version 10.0.19044 Compilation 19044

- Node version (run `node -v`): v16.13.0
- auto-launch version: 6.0.2
- The options you're passing to auto-launch: 
```
let autoLaunch = new AutoLaunch({
      name: "Name of my app"
 });
```
- Are you using NW.js or Electron?
   Electron
  - If so, which version?
  16.0.6
  - If you're using Electron, are you using auto-launch from the main process or a renderer process?
  Main process on the function `app.whenReady().then(() => ...)`
- Can you reproduce this with a simple Hello World example app?
  Basically I use the function inside the `whenReady()` function callback. When I start the system, the app doesn't work.

---

I am using the basic usage of the library. The users of my app use Windows and the app is not opening when the system startup.

I open the Task Manager and see if the app is on the "Initialize" tab. And it is but for some reason it is not opening.

**Comment by gschafra** [comment]
Also happens here with same specs
